src/components/Admin/Authentication/Util.tsx

```diff
@@ -112,16 +118,17 @@ const PasswordFormInput: ({
   handlePasswordVisibility,
   error,
 }: PasswordFormInputProps) => JSX.Element = ({
-  label = 'Password',
+  label = 'password',
```

Comment:
Change this variable name to ```labelKey``` or something of the like now that it no longer represents what will actually be displayed on the screen